### PR TITLE
Add changelog and upgrade

### PR DESCRIPTION
**Pull request by notFloran**
#9

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG
+
+## 4.0.0
+
+* UID feature is now called Request id (#4)
+* Rename namespace from Hexanet/Common/ to TheRedDot/ (#12)
+* Rename ApacheUniqueIdProvider to ServerRequestIdProvider (#16)
```
Comment (nclsHart) [inline CHANGELOG.md:7]: Maybe we should also list classes marked as final.

File: UPGRADE-4.0.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# UPGRADE to 4.0
+
+* Rename bundle class from `Hexanet\Common\MonologExtraBundle\HexanetMonologExtraBundle` to `TheRedDot\MonologExtraBundle\TheRedDotMonologExtraBundle`
+* Rename the package config from `hexanet_monolog_extra` to `the_red_dot_monolog_extra`
+* The UID feature has been renommed to Request-id
```
Comment (nclsHart) [inline UPGRADE-4.0.md]:
```suggestion
* The UID feature has been renamed to Request-id
```

File: UPGRADE-4.0.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Before:
+
+```
```
Comment (nclsHart) [inline UPGRADE-4.0.md]: yaml

File: UPGRADE-4.0.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+hexanet_monolog_extra:
+    processor:
+        uid: true
+    logger:
+        add_uid_to_response: true
+```
+
+After:
+
+```
```
Comment (nclsHart) [inline UPGRADE-4.0.md]: yaml

**Review by nclsHart — CHANGES_REQUESTED** [review]

File: UPGRADE-4.0.md (hunk 2 above)
Comment (notFloran) [reply UPGRADE-4.0.md]: Franglish world !

**Review by nclsHart — APPROVED** [review]

**Pull request merged**